src/app/components/ScrollablePromo/index.jsx

```diff
@@ -41,21 +61,57 @@ const ScrollablePromo = ({ blocks, blockGroupIndex }) => {
     return null;
   }
 
-  const blocksWithoutTitle = blocks[0].type === 'title' ? tail(blocks) : blocks;
+  const blocksWithoutTitle =
+    blocks[0]?.type === 'title' ? tail(blocks) : blocks;
 
   const isSingleItem = blocksWithoutTitle.length === 1;
 
+  const title = pathOr(
+    'You may also be interested in',
+    ['recommendationTitle'],
+    translations,
+  );
+
+  const labelId = 'recommendations-heading';
+  const a11yAttributes = {
+    as: 'section',
+    role: 'region',
+    'aria-labelledby': labelId,
+  };
+
   return (
-    <GridItemMediumNoMargin>
+    <GridItemMediumNoMargin
+      {...(recommendations && { 'data-e2e': { labelId }, ...a11yAttributes })}
+    >
+      {recommendations && (
+        <LabelComponent
+          script={script}
+          service={service}
+          dir={dir}
+          labelId={labelId}
```

Comment:
Might need to be more specific than 'recommendations-heading' could be somethiing similar on the page?

Reply:
This is what we currently use for the live recommendation (control version), but I will make sure it gives out no problems.